<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Experiment</class>
 <widget class="QMainWindow" name="Experiment">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>498</width>
    <height>423</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="stimuliTab">
       <attribute name="title">
        <string>Stimuli</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout">
        <item row="0" column="0">
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <item>
             <widget class="QLabel" name="collectionFileLabel">
              <property name="text">
               <string>Collection File</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="collectionFileLineEdit"/>
            </item>
            <item>
             <widget class="QToolButton" name="openCollectionFileButton">
              <property name="text">
               <string>&amp;Open</string>
              </property>
              <property name="shortcut">
               <string>Ctrl+O</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="donePushButton_2">
              <property name="text">
               <string>&amp;Done</string>
              </property>
              <property name="shortcut">
               <string>Ctrl+D</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tagsTab">
       <attribute name="title">
        <string>Tags</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="0" column="0">
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QPushButton" name="addTagPushButton">
            <property name="text">
             <string>&amp;Add Tag</string>
            </property>
            <property name="shortcut">
             <string>Ctrl+A</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="removeTagPushButton">
            <property name="text">
             <string>&amp;Remove Tag</string>
            </property>
            <property name="shortcut">
             <string>Ctrl+R</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>118</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QPushButton" name="donePushButton">
            <property name="text">
             <string>&amp;Done</string>
            </property>
            <property name="shortcut">
             <string>Ctrl+D</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="1" column="0">
         <widget class="QTableView" name="editTagsView">
          <property name="selectionMode">
           <enum>QAbstractItemView::SingleSelection</enum>
          </property>
          <property name="selectionBehavior">
           <enum>QAbstractItemView::SelectRows</enum>
          </property>
          <attribute name="horizontalHeaderStretchLastSection">
           <bool>true</bool>
          </attribute>
          <attribute name="verticalHeaderVisible">
           <bool>false</bool>
          </attribute>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
